docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -138,7 +367,54 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use atparse-formatted template file ``atparse.txt`` with content
+
+.. code:: sh
+  @[greeting], @[recipient]!
+
+* Convert an atparse-formatted template file to Jinja2 format:
+
+  .. code:: sh
+
+    $ uw config translate --input-file atparse.txt --input-format atparse --output-format jinja2
+    {{greeting}}, {{recipient}}!
+
+  Shell redirection via ``|``, ``>``, et al may also be used to stream output to a file, another process, etc.
+
+* Convert the template to a file via command-line argument:
+
+  .. code:: sh
+
+    $ uw config realize --input-file values.yaml --output-file realized.yaml
+
+  The content of ``jinja2.txt``:
+
+  .. code:: sh
+
+    {{greeting}}, {{recipient}}!
+
+* With the ``--dry-run`` flag specified, nothing is written to ``stdout`` (or to a file if ``--output-file`` is specified), but a report of what would have been written is logged to ``stderr``:
+
+  .. code:: sh
+
+    $ uw config translate --input-file atparse.txt --input-format atparse --output-format jinja2 --dry-run
+    [2024-01-03T16:41:13]     INFO {{greeting}}, {{recipient}}!
+
+
+* If an input file is read alone from ``stdin``, ``uw`` know how to parse its content as we must always specify the formats:
```

Comment:
`know how` -> `will not know how` ? Maybe
> If the input is read from `stdin`, `uw` will not know how to parse it, so the format must be specified:

? Also, it's not necessarily a "file" that's being read -- it could be the output of another process. Though it obviously is a file in this example, it's easy enough to not mention it.

Reply:
Ah, missed that, I think I've cleaned it up